src/commands/InitCommand.js

```diff
@@ -54,6 +54,7 @@ export default class InitCommand extends Command {
 
   execute(callback) {
     this.ensurePackageJSON();
+    this.ensurePackagesDir();
```

Comment:
This needs to be called _after_ `ensureLernaJson`, otherwise `this.repository.lernaJson` might not exist.